source/osh/datareceiver/VideoH264.js

```diff
@@ -60,8 +60,8 @@ class VideoH264 extends DataSource {
      */
     parseTimeStamp(data) {
         // read double time stamp as big endian
-        this.lastTimeStamp = new DataView(data).getFloat64(0, false) * 1000;
```

Comment:
In this case it should be lastTimeStamp and not lastReceiveTime, because here we're not using wall clock time but parsing the timestamp that's in the data.

Same for all data receivers below.